### ISSUE TITLE
Version 0.7.2 has not been uploaded to anaconda.org

### ISSUE THREAD

**Issue by sdvillal** [conda-forge/jpype1-feedstock#15]
Maybe we need to rerun the CI jobs manually?

**Comment by marscher** [comment]
I think we cannot rerun them manually. It seems the merge jobs have not been created at all. But the pipeline changes so frequently I cannot tell for sure.

The log showed this message:

```
2020-03-01T13:50:36.1005844Z Distribution /home/conda/feedstock_root/build_artifacts/linux-64/jpype1-0.7.2-py36hc9558a2_0.tar.bz2 is new for conda-forge, but no upload is taking place because the BINSTAR_TOKEN is missing. 
```

Not a single build e.g. platform is available for 0.7.2

I guess we need to contact @conda-forge/team


Am 7. März 2020 16:35:07 MEZ schrieb Santi Villalba <notifications@github.com>:

**Comment by marscher** [comment]
packages are now available
